autoload/tagbar.vim

```diff
@@ -3162,7 +3163,7 @@ function! s:ExecuteCtags(ctags_cmd) abort
         call tagbar#debug#log('Exit code: ' . v:shell_error)
         redraw!
     else
-        let py_version = get(g:, 'tagbar_python', 1)
+        let py_version = get(g:, 'tagbar_python', 0)
```

Comment:
I would prefer we don't change the default behavior for the `g:tagbar_python` variable. Initializing the stdin to DEVNULL is fine, but can we verify if setting the default for `g:tagbar_python = 0` is really necessary? Would it be possible to have this as a user setting so we aren't changing the default? We can add documentation around this variable to indicate when it should be set to 0.

Reply:
Unluckily `g:tagbar_python = 1` reverts to the same problem.  

Tried it with gvim9.0.1672_64 on windows 11 (`> winget install --id vim.vim`), with python3.11.4. [ctags5.8](https://sourceforge.net/projects/ctags)

![image](https://github.com/preservim/tagbar/assets/29801096/80c51be5-1afc-4178-971a-c6753ccd7c31)

Reply:
You are still seeing this after you set the `let g:tagbar_python = 0` in your vimrc? That should set it to 0 for you and it won't use the default.

Reply:
That works! 
Guess a install for windows users hint should be in place? Like

Windows users should add `let g:tagbar_python = 0` to their `vimrc`

Reply:
I am not following closely enough to suggest an exact solution here, but somehow or another if we're at a point of recommend all Windows users do something special we should just make that the default for Windows. Or whatever the situation, the out of box configuration for a given platform should be the best defaults for that platform without most people needing to set something special.